config/routes.rb

```diff
@@ -6,6 +6,7 @@
     resources :categories
   end
   get 'static_pages/home'
+  post 's3/sign'
```

Comment:
@barancw quick question: is there a reason this exposes POST and not GET? 
It looks like we can make the client component work with either but the example specifies GET:
```
<ReactS3Uploader
    signingUrl="/s3/sign"
    signingUrlMethod="GET"
```
_(From https://github.com/odysseyscience/react-s3-uploader)_